### PR TITLE
Bookstore UI

### PR DESCRIPTION
**Pull request by henask12**
# In This Pull Request

### This pull request adds Tailwind CSS integration and creates reusable components with React Router V6 for the application. The following changes have been made:

- [x] Created Reusable Components: Created the following reusable components inside the /components directory:

- Book.js: Contains individual book state, as well as a button to delete the book.
- BookList.js: Contains a list that renders individual books.
- BookForm.js: Contains a form to create new books.
- Navigation.js: Contains a navigation bar with links to the two routes created using React Router V6.

- [x] Created Tailwind Classes in Components: Updated the components to use Tailwind CSS classes for styling. Styles were added to the Book.js and App.js components to make them visually appealing.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+              </Link>
+            </li>
+            <li>
+              <Link to="/catagories" className="text-white hover:text-teal-200">
+                Categories
+              </Link>
+            </li>
+          </ul>
+        </div>
+      </nav>
```
Comment (Lucas-Masaba) [inline src/App.js]: [OPTIONAL]
- I would suggest making this navbar you created here a component and importing it here. I also noticed you created a Navigation component but you did not use it here.

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      <Routes>
+        <Route exact path="/" element={<Home />} />
+        <Route exact path="/catagories" element={<Categories />} />
```
Comment (Lucas-Masaba) [inline src/App.js]: - [ ] As mentioned in the requirements, the Categories component should be added to your components folder. Currently, I see you just added it to this component which is not required.

Comment (Lucas-Masaba) [inline src/App.js]: - [ ] As mentioned in the requirements, this should be a route to the Book component and it should be in the components folder. Currently, I see you have it in this component.

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @henask12 👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- Linter checks passing :heavy_check_mark: 
- Descriptive README :scroll: 
- Well written commit messages :writing_hand: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi Hena, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good job following the instructions of the previous CR.
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**